packages/frontend-2/assets/css/tailwind.css

```diff
@@ -77,3 +81,8 @@ body,
 div#__nuxt {
   height: 100%;
 }
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial,
```

Comment:
Is this correct? I'm not seeing the fonts imported at the top of the file here. Also I'm concerned about setting the main/highest priority font to be something Apple specific that might not work on other operating systems, this could make fonts differ across OSs making it more difficult for us to ensure a comparable experience for everyone

Reply:
adding it as a separate class, and pref through tailwind config!

Reply:
tried a thing, tbd if the best way 😅